drivers/ad5686/ad5686.h

```diff
@@ -113,9 +108,21 @@ enum comm_type {
 	I2C,
 };
 
+enum ad5686_dac_channels {
+	AD5686_CH_0 = 0,
+	AD5686_CH_1,
+	AD5686_CH_2,
+	AD5686_CH_3,
+	AD5686_CH_4,
+	AD5686_CH_5,
+	AD5686_CH_6,
+	AD5686_CH_7,
+};
+
 struct ad5686_chip_info {
 	uint8_t		resolution;
 	enum comm_type	communication;
+	const unsigned int *channel_addr;
```

Comment:
Use fixed width integers to avoid ambiguity. 'const uint32_t' instead of 'const unsigned int'.